client/src/App.js

```diff
@@ -1,39 +1,30 @@
-import './App.css';
 import React, { useState } from 'react';
-/*import Name from './Components/Header'*/
+import './App.css';
+import HGIB from './Components/Header';
+import DropDown from './Components/DropDown';
+import SearchButton from './Components/button';
+import ResultBox from './Components/ResultBox';
+
 
 function App() {
   const [product, setProduct] = useState();
-
   return (
     <body className="body">
       <form>
-        <h1>What would You like to give back?</h1>
-        <select
-          className="custom-select"
-          id="SelectID"
-          value={product}
-          onChange={e => setProduct(e.target.value)}
-        >
-          <option>Select an Option</option>
-          <option>Plastic Transparent</option>
-          <option>Plastic Black</option>
-          <option>Aluminum</option>
-          <option>Aluminum Contaminated</option>
-          <option>Paper Contaminated</option>
-          <option>Paper Coated</option>
-        </select>
-        <br></br>
+        <HGIB />
+        <h1 className="question">What would You like to give back?</h1>
+        <DropDown />
```

Comment:
pleas do not use `br` for spacing, try to use CSS instead.

Reply:
Agree. br is a bit too much "hands on"...styles and beauty will be postponed until the desired functionalities for the capStoneFinishingLine are set . Make it work, than efficient and than beautiful, as coaches taught us :). So definitely on the radar.